docs/ajaxApi.rst

```diff
@@ -622,7 +622,7 @@ This API executes a flow via an ajax call, supporting a rich selection
 of different options. Running an individual job can also be achieved via
 this API by disabling all other jobs in the same flow.
 
--  **Method:** GET
+-  **Method:** POST
 -  **Request URL:** /executor?ajax=executeFlow
 -  **Parameter Location:** Request Query String
```

Comment:
Request `body` maybe?

Reply:
Thanks for fixing the docs for the login endpoint, however, we also need to specify "Request body" instead of "Request query string" for the execute flow endpoint. 
In addition, we should fix the uris of both endpoints as parameters go in the request body for POST requests:
Instead of `/?action=login` it should be just `/`
Instead of  `/executor?ajax=executeFlow` it should be `/executor`

I'll make the change myself in a separate PR.